ingestion/src/metadata/ingestion/ometa/mixins/user_mixin.py

```diff
@@ -39,6 +39,29 @@ class OMetaUserMixin:
         + ES_INDEX_MAP[User.__name__]
     )
 
+    # Allow for more flexible name lookup
+    name_search = (
```

Comment:
@pmbrull there is fieldQuery api as well, where you can pass fieldName and fieldValue in this case name=User.name will get you exact match result

Reply:
re the `name=User.name` wanted to keep it same as the UI is doing when searching for users. E.g., if the user name is random.user but it is defined in Airflow as Random User , if I used the exact match I think it was not giving me back the record